components/mysql/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/mysql",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

I don't think this is needed since we're not touching `mysql.app.mjs`, right?
```suggestion
  "version": "2.2.0",
```